src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/ReferenceDataController.kt

```diff
@@ -33,6 +41,8 @@ class ReferenceDataController(
 
 // TODO: migrate name, label to id, name to be closer to e.g. courts and prisons responses
 data class LocalDeliveryUnitResponse(val name: String, val label: String)
+data class MappaLevelResponse(val name: String, val label: String)
```

Comment:
Per the above // TODO I think we agreed to use (id, name) and not (name, label) as properties.

Reply:
can do, but will break UI.

Reply:
also, courts and prisons aren't Enums, 🤷‍♂️

Reply:
changed to be consistent with rest of controller. LDU will need future amendment